srcpkgs/runit-void/files/09-apparmor.sh

```diff
@@ -18,15 +18,9 @@ if [ -n "$APPARMOR" ]; then
 	[ "$APPARMOR" = "complain" ] && AACOMPLAIN="-C"
 
 	if [ -d /etc/apparmor.d -a -x /usr/bin/apparmor_parser ]; then
-		for profile in /etc/apparmor.d/*; do
-		case "$profile" in
-			*.new-*_*) continue ;;
-		esac
-		if [ -f "$profile" ]; then
-			printf '* Load profile %s: %s\n' "($APPARMOR)" "$profile"
-			apparmor_parser -a $AACOMPLAIN "$profile"
-		fi
-		done
+		find /etc/apparmor.d -maxdepth 1 -type f ! -name '*.new-*_*' |\
+			xargs -d"\n" -I{} --max-procs="$(nproc)" \
+				sh -c "printf '* Load profile %s: %s\n' '($APPARMOR)' '{}'; apparmor_parser -a $AACOMPLAIN '{}'"
```

Comment:
Allows command injection. I'm not considering this an attack vector, as the files are root writeable, nevertheless, avoid this.

Reply:
Command injection? You mean shell spawning or in general? I don't see command injection here. I mean, even $APPARMOR without anything else in a shell script it can be a command injection if you don't control the source.